pyintegration/deephaven2/_wrapper_abc.py

```diff
@@ -14,4 +14,9 @@ class JObjectWrapper(ABC):
     @property
     @abstractmethod
     def j_object(self) -> jpy.JType:
-        ...
\ No newline at end of file
+        ...
+
+    @classmethod
+    @abstractmethod
+    def j_object_type(cls) -> jpy.JType:
+        ...
```

Comment:
I'm wondering if there are any helper constructors, or we should be expecting the impl classes to be setting values on the class instead. That would allow us to have stronger checks:

```python
isinstance(cls.j_object_type(), type) == True
ininstance(x.j_object(), cls.j_object_type()) == True
```